client/src/pages/Login.jsx

```diff
@@ -71,7 +71,7 @@ const Login = () => {
   return loading ? (
     <p>Loading...</p>
   ) : (
-    <div className="flex flex-col items-center justify-center min-h-screen bg-gray-100">
+    <div className="flex flex-col items-center justify-center min-h-screen bg-gray-100 text-black">
```

Comment:
The text background is also black
```suggestion
    <div className="flex flex-col items-center justify-center min-h-screen bg-gray-100">
```